### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by RahulSingh9131**
1. simple navigation
simpleNavigation
2. landing page navigation
LandingPageNavigation

### CODE REVIEW

File: docs/navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    <main>
+        <div class="docs-wrapper">
+            <header>
+                <div class="container">
+                    <div class="logo">
+                        <img src="../../assets/brand-logo.svg" alt="Logo">
+                        <h3>Pegasus</h3>
+                    </div>
+                    <nav class="links">
+                        <ul>
```
Comment (shuntzhere) [inline docs/navigation/navigation.html:23]: You can give the container class to header itself.

File: css/components/navigation.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    width: 100%;
+    align-items: center;
+    padding: 1.5rem .7rem;
+    justify-content: space-between;
+    background-color: #333;
+}
+.left h5{
+    font-size: var(--normal-font-size);
+    color: #fff;
+}
```
Comment (shuntzhere) [inline css/components/navigation.css:14]: Avoid fixed color values.

**Review by shuntzhere — COMMENTED** [review]
Code looks good, suggested some minor changes for maintaining cleaner code.
Also check if you can create base classes for recurring lines of code.

File: docs/navigation/navigation.html (hunk 1 above)
Comment (RahulSingh9131) [reply docs/navigation/navigation.html:23]: yes i can , but I have given a max-width to that container class and used this in other places too. So to make the code semantically correct I have separately added header tag.

File: css/components/navigation.css (hunk 2 above)
Comment (RahulSingh9131) [reply css/components/navigation.css:14]: sure will create a variable in color palette.

File: docs/navigation/navigation.html (hunk 1 above)
Comment (shuntzhere) [reply docs/navigation/navigation.html:23]: Got it, makes sense now.

File: css/components/navigation.css (hunk 2 above)
Comment (shuntzhere) [reply css/components/navigation.css:14]: Yeah that would do it.

**Review by shuntzhere — APPROVED** [review]
Everything looks good, just make sure to maintain proper spacing and use extensions like prettier to indent the code properly.

File: css/components/navigation.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+.middle{
+    position: relative;
+    display: flex;
+    align-items: center;
+}
+.middle input{
+    padding: .5rem;
+    width: 15rem;
+    outline: none;
```
Comment (sadathshariff) [inline css/components/navigation.css]: try not to use outline: none as it creates issues for people navigating the web with a keyboard.

File: css/components/navigation.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    display: flex;
+    width: 100%;
+    align-items: center;
+    padding: 1.5rem .7rem;
+    justify-content: space-between;
+    background-color: #333;
+}
+.left h5{
+    font-size: var(--normal-font-size);
+    color: #fff;
```
Comment (sadathshariff) [inline css/components/navigation.css:13]: try using color variable here

File: css/components/navigation.css (hunk 3 above)
Comment (RahulSingh9131) [reply css/components/navigation.css]: sure , i just removed that.

**Review by sadathshariff — COMMENTED** [review]
Just a small change, rest everything looks good

**Review by sadathshariff — APPROVED** [review]

**Pull request merged**